src/renderer/features/governance/ReferendumList/model/referendum-list-model.ts

```diff
@@ -35,8 +33,9 @@ const requestOffChainReferendumsFx = createEffect(
   },
 );
 
-const getVotesFx = createEffect((api: ApiPromise): Promise<any[]> => {
-  return governanceService.getVotesFor(api, '12mP4sjCfKbDyMRAEyLpkeHeoYtS5USY4x34n9NMwQrcEyoh');
+const getVotesFx = createEffect((api: ApiPromise): Promise<Record<TrackId, Voting>> => {
+  return governanceService.getVotingFor(api, '12mP4sjCfKbDyMRAEyLpkeHeoYtS5USY4x34n9NMwQrcEyoh');
+  // return governanceService.getVotingFor(api, '153YD8ZHD9dRh82U419bSCB5SzWhbdAFzjj4NtA5pMazR2yC');
```

Comment:
Do you need this commented code? Maybe create variable for this address?

Reply:
Testing address for POC, will be removed as soon as I will start working on Referendum list (today)